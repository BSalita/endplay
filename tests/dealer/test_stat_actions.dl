produce 20

action 
  average "points " hcp(north),
  frequency "hcp(north)" (hcp(north), 3, 26),
  frequency "hcp(north) vs hcp(south) in the range 5-15" (hcp(north), 5, 15, hcp(south), 5, 15)