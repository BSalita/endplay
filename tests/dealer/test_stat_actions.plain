
points 9.9

