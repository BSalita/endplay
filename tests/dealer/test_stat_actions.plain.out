
points 8.85

